arangodb-net-standard/PregelApi/Models/PregelJobGssStatusItem.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace ArangoDBNetStandard.PregelApi.Models
+{
+    public class PregelJobGssStatusItem
+    {
+        public int? VerticesProcessed { get; set; }
+        public int? MessagesSent { get; set; }
+        public int? MessagesReceived { get; set; }
+        public long? MemoryBytesUsedForMessages { get; set; }
```

Comment:
Are these properties optional because they are not always returned in the response? The online doc doesn't make any mention of this fact, so just looking to confirm. https://www.arangodb.com/docs/devel/http/pregel.html#get-pregel-job-execution-status

Reply:
Ah, never mind. When a job is not finished, these won't be returned.